src/QA/QA.Web/ViewModels/WithdrawVacancy/AcknowledgeEditModelValidator.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentValidation;
+
+namespace Esfa.Recruit.Qa.Web.ViewModels.WithdrawVacancy
+{
+    public class AcknowledgeEditModelValidator : AbstractValidator<AcknowledgeEditModel>
+    {
+        public AcknowledgeEditModelValidator()
+        {
+            RuleFor(x => x.Acknowledged)
+                .Equal(true)
```

Comment:
Should this be false? Haven't run it.

Reply:
It must be true in order to be valid otherwise it creates the validation error